### ISSUE TITLE
24h2 ETA

### ISSUE THREAD

**Issue by drex96** [fabianosrc/TermsrvPatcher#9]
Sorry to bother, when do you think support for 24h2 is coming? Thanks!

**Comment by fabianosrc** [comment]
@drex96 not a problem, bro. I think it'll take a while. Microsoft made some significant changes to the termsrv.dll structure that I can't break.

**Comment by viethvq** [comment]
Any fixed already with 24h2, bro?

**Comment by viethvq** [comment]
> @drex96 not a problem, bro. I think it'll take a while. Microsoft made some significant changes to the termsrv.dll structure that I can't break.

Any fixed already with 24h2, bro?

**Comment by fabianosrc** [comment]
Hi @viethvq, not for now. I'm still working on a solution.

**Comment by rajuddinshaikh** [comment]
Tried using the script after installing Windows 11 24H2 update and did not work for me either.

**Comment by rajuddinshaikh** [comment]
Tried using the script after installing Windows 11 24H2 update and did not work for me either.

**Comment by ccwang65** [comment]
Any hope? I revert to rdpwrap.dll and .ini, that's working but windows defender marked the dll as severe threat. Thereful might not allowed by company.

**Comment by fabianosrc** [comment]
Hi @ccwang65  sorry, for a while I can't to fix this. rdpwrap works but it really seems to be something not very reliable for Windows.

**Comment by wherefrom** [comment]
> Any hope? I revert to rdpwrap.dll and .ini, that's working but windows defender marked the dll as severe threat. Thereful might not allowed by company.

does rdpwrap work on 24H2?  I tried but failed, rdpwrapper says "fully supported" but unable to multi remotely login.

**Comment by ccwang65** [comment]
> does rdpwrap work on 24H2? I tried but failed, rdpwrapper says "fully supported" but unable to multi remotely login.
Yes it works. You can try the original termsrv.dll, not the patched one,and the right rdpwrap.ini (https://github.com/affinityv/INI-RDPWRAP)

**Comment by ccwang65** [comment]
Good news, this is working:
(from https://forums.mydigitallife.net/threads/discussion-windows-11-termsrv-dll-patching.83813/page-3)

Windows 11 Pro
Version: 24H2
OS Build: 26100.2454

===============================
termsrv.dll (x64) 10.0.26100.2454
===============================

Search:
8B 81 38 06 00 00 39 81 3C 06 00 00 75
Replace:
B8 00 01 00 00 89 81 38 06 00 00 90 EB

**Comment by wherefrom** [comment]
> Good news, this is working: (from https://forums.mydigitallife.net/threads/discussion-windows-11-termsrv-dll-patching.83813/page-3)
> 
> Windows 11 Pro Version: 24H2 OS Build: 26100.2454
> 
> # ===============================
> termsrv.dll (x64) 10.0.26100.2454
> Search: 8B 81 38 06 00 00 39 81 3C 06 00 00 75 Replace: B8 00 01 00 00 89 81 38 06 00 00 90 EB

Great, it works!

**Comment by viethvq** [comment]
Hey Fabianosrc, already have patern for 24H2 ...waiting for your update =))

**Comment by fabianosrc** [comment]
Hello my friend, please wait a little as I will make the necessary adjustments this week.

**Comment by ccwang65** [comment]
> Hey Fabianosrc, already have patern for 24H2 ...waiting for your update =))

I have create an c# exe tool. My trick is ssh to local admin account, copy it on home directory and run it with no parameter, and restart termservice. If you don't want to open ssh server, copy termsrv.dll to home, run exe with the dll file as parameter, it will be updated. Then you have to copy it back to replace original one. Use fc.exe /b to compare old and new to find the differences.

**Comment by fabianosrc** [comment]
Hi there. Windows 11 24H2 supported now! Thank you so much.
